src/main/java/com/werfen/report/test/utils/model/FileComparision.java

```diff
@@ -0,0 +1,25 @@
+package com.werfen.report.test.utils.model;
+
+public class FileComparision {
```

Comment:
This new class and approach of using a Comparison class looks good to me. My only concern is a possible future need of partial comparisons of files. For instance, just check a sheet on an Excel workbook. How this would be approached? With another other class (i.e SheetComparison) or maybe renaming this one to something more generic (i,e ReportComparison)?

Reply:
Totally right. If you look at the class it is so generic and nothing is related to files: it is simply a flag and a text.

Reply:
Looks good to me now. Thanks Alfons.